client/src/templates/Challenges/redux/selectors.js

```diff
@@ -31,7 +31,8 @@ export const successMessageSelector = state => state[ns].successMessage;
 export const projectFormValuesSelector = state =>
   state[ns].projectFormValues || {};
 
-export const portalDocumentSelector = state => state[ns].portalDocument;
+export const portalDocumentSelector = state => state[ns].portalWindow?.document;
+export const getPortalWindow = state => state[ns].portalWindow;
```

Comment:
```suggestion
export const portalWindowSelector = state => state[ns].portalWindow;
```
To keep to the convention.